cloudcompchem/utils.py

```diff
@@ -27,7 +27,12 @@ def run_dft_calculation(dft_input: DFTRequest) -> SinglePointEnergyResponse:
     else:
         calc = dft.RKS(mol)
     calc.xc = dft_input.functional
-    energy = calc.kernel()
+    calc.kernel()
```

Comment:
yes, it still returns a float, but doesn't return everything else (e.g. orbital stuff). since we have to extract the other info from the mutated state, i figured it was better to be consistent and take the total energy from the calculation state as well.